clients/frontend-authorization/src/oidc/browser/BrowserAuthorizationClient.ts

```diff
@@ -244,7 +247,8 @@ export class BrowserAuthorizationClient extends BrowserAuthorizationBase<Browser
       this._accessToken = undefined;
       return;
     }
-    this._accessToken = AccessToken.fromTokenResponseJson(user, user.profile);
+    this._accessToken = `Bearer ${user.access_token}`;
+    this._expiresAt = new Date(user.expires_at * 1000);
```

Comment:
how does `getAccessToken` result in a token refresh below?

Reply:
It is not currently implemented as `master`  does not implement it and we didn't change the way it works in this PR. @pbell97 will work on an update to the `getAccessToken` method that refreshes the token and checks the expiry.

Reply:
OK, but i'd like to make refresh happen automatically (on a timer based on expiry.)

Reply:
Token refresh already does happen on a timer in `BrowserAuthorizationClient`.
The timer is managed by the underlying `UserManager`, which refreshes the token 1 minute before it expires by default (we currently accept this default)

When the timer is up, a silent refresh is attempted, and if it's successful, the token is passed into `initAccessToken` by way of `_onUserLoaded`, which is hooked into an event raised by the `UserManager`

Reply:
Awesome, thanks @evelynpreslar-bentley. I felt like I was missing something there but wasn't sure what :)